.github/workflows/integration-tests.yml

```diff
@@ -470,7 +507,7 @@ jobs:
       - name: Install brew dependencies
         run: |
           brew update
-          brew install ccache llvm@19 lld
+          brew install ccache llvm@19 lld coreutils
```

Comment:
`coreutils` is needed for `sha256sum` which is needed to compute the cache keys.